src/apps/activities/domain/response_values.py

```diff
@@ -329,6 +337,34 @@ class AudioPlayerValues(PublicModel):
     file: str | None = Field(default=None)
 
 
+class _PhrasalTemplateTextField(PublicModel):
+    type: PhrasalTemplateFieldType = PhrasalTemplateFieldType.SENTENCE
+    text: str
+
+
+class _PhrasalTemplateItemResponseField(PublicModel):
+    type: PhrasalTemplateFieldType = PhrasalTemplateFieldType.ITEM_RESPONSE
+    id: str | None = None
+    display_mode: str
+
+
+class _PhrasalTemplateLineBreakField(PublicModel):
+    type: PhrasalTemplateFieldType = PhrasalTemplateFieldType.LINE_BREAK
+
+
+PhrasalTemplateField = _PhrasalTemplateTextField | _PhrasalTemplateItemResponseField | _PhrasalTemplateLineBreakField
+
+
+class PhrasalTemplatePhrase(PublicModel):
+    image: str | None = Field(default=None)
+    fields: list[PhrasalTemplateField]
+
+
+class PhrasalTemplateValues(PublicModel):
+    type: Literal[ResponseType.PHRASAL_TEMPLATE] | None
+    phrases: list[PhrasalTemplatePhrase]
```

Comment:
I think we need a field for the [card title](https://www.figma.com/design/3wcZ6bxK2QVo733ADxVjBw/SSI-2024?node-id=564-68081&t=Ai9jbXx3DiU4zTpW-4) too
```suggestion
    card_title: str
    phrases: list[PhrasalTemplatePhrase]
```

Reply:
I other item type, it uses item name, but design creates another field for it, so I will add it

Reply:
added